Sources/docc/DocCDocumentation.docc/customizing-the-appearance-of-your-documentation-pages.md

```diff
@@ -79,7 +79,26 @@ of a green hue.
 
 As a general rule, the default color properties provided by DocC assumes a
 naming convention where "fill" colors are used for backgrounds and "figure"
-colors are used for foreground colors, like text.
+colors are used for foreground colors, like text. Note that colors defined in 
+`theme-settings.json` will be used across all pages of your documentation bundle.
+
+**Changing a color on for a specific article page**
```

Comment:
Should this be a sub-sub heading that briefly describe the per-page customization? That could be a good place to talk about both `PageColor` and `PageImage` together.